fail2ban/tests/actiontestcase.py

```diff
@@ -71,6 +71,12 @@ def testSubstituteRecursiveTags(self):
 									'ABC': '123 192.0.2.0',
 									'xyz': '890 123 192.0.2.0',
 								})
+		# obscure embedded case
+		self.assertEqual(CommandAction.substituteRecursiveTags({'A': '<<PREF>HOST>', 'PREF': 'IPV4'}),
+						 {'A': '<IPV4HOST>', 'PREF': 'IPV4'})
```

Comment:
<blockquote>who to implement support for IPv6 while avoiding duplication</blockquote>
To avoid duplication we can use normal substitution `%(parameter)s` without any recursion.
After #868 we can additionaly use `%(known/parameter)s` even across a config files.